controller/db/api_session_store.go

```diff
@@ -149,44 +151,43 @@ func (store *apiSessionStoreImpl) onEventualDelete(db boltz.Db, name string, api
 		}).Error("error querying for session associated to an api session during onEventualDelete")
 	}
 
-	for _, id := range idCollector.ids {
-		changeContext := change.New().SetSourceType("events.emitter").SetChangeAuthorType(change.AuthorTypeController)
-		err = db.Update(changeContext.NewMutateContext(), func(ctx boltz.MutateContext) error {
-			if err := store.stores.session.DeleteById(ctx, id); err != nil {
-				if boltz.IsErrNotFoundErr(err) {
-					return nil
-				}
-				return err
-			}
-			return nil
-		})
-
-		if err != nil {
-			pfxlog.Logger().WithError(err).WithFields(map[string]interface{}{
-				"eventName":    name,
-				"apiSessionId": string(apiSessionId),
-				"sessionId":    id,
-			}).Error("error deleting for session associated to an api session during onEventualDelete")
-		}
+	if store.stores.rateLimiter.GetQueueFillPct() > 0.5 {
+		time.Sleep(time.Second)
```

Comment:
This seems odd. Intentional?

Reply:
Yes, I did encounter a case where the background processing of API sessions seemed to cause an issue. 
On the other, unbeknownst to me at the time, I had exhausted my I/O burst capacity on the disk. 
So on the one hand this shouldn't hurt anything. It just slows down background processing if there's a lot of foreground activity happening. On the other, it might not be necessary. I can take it out or leave it.